scripts/testnet/include/config.sh

```diff
@@ -221,6 +246,13 @@ updateTxGenConfig() {
   popd
 }
 
+updateSovereignTxGenConfig() {
+  updateTxGenConfig
+  pushd $TESTNETDIR/txgen/config/nodeConfig/config
+  updateTOMLValue enableEpochs.toml "SCDeployEnableEpoch" 0
+  updateTOMLValue enableEpochs.toml "BuiltInFunctionsEnableEpoch" 0
+  updateTOMLValue enableEpochs.toml "ESDTEnableEpoch" 0
```

Comment:
I think this func could be removed.
We can set these flags to 0 from `enableEpochs.toml`

Reply:
yes, but it should be set to 0 there... some of the flags could not be set to 0 and this is why an update at start time was needed